Dockerfile

```diff
@@ -21,34 +22,35 @@ RUN apt update -y --fix-missing && \
 ADD https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh /miniconda.sh
 RUN sh /miniconda.sh -b -p /conda && /conda/bin/conda update -n base conda
 ENV PATH=${PATH}:/conda/bin
-
 # Enables "source activate conda"
 SHELL ["/bin/bash", "-c"]
 
-# Combined libgdf/pygdf conda env
-ADD conda_environments/gdf_build.yml /conda_environments/gdf_build.yml
-# Also tested working with 3.5
+# Build combined libgdf/pygdf conda env
 ARG PYTHON_VERSION=3.6
-RUN conda env create -n gdf python=${PYTHON_VERSION} --file /conda_environments/gdf_build.yml
+RUN conda create -n gdf python=${PYTHON_VERSION}
+RUN conda install -n gdf -y -c numba -c conda-forge -c defaults \
+      numba \
+      pandas
```

Comment:
We need to lock pandas to a specific version. I believe we're only compatible with 0.20.3 currently.

Reply:
https://github.com/gpuopenanalytics/pygdf/blob/master/Dockerfile#L50 should be changed to "make -j$CPU_COUNT install"

Reply:
In my own testing, DataFrame.from_pandas and DataFrame.to_pandas both work with PyGDF master. Is there a known compat issue for Pandas >0.20.3?

Would like to hear other opinions on this, but since this is a build/dev container, I personally would want to build and against current numba and pandas packages, and work towards finding and fixing issues.

For known compat issues, I'm happy to lock it down and leave a comment with a link to the blocking issue.

Reply:
I believe there was categorical issues with Pandas versions newer than 0.20.3 but I could be mistaken.

Reply:
@kkraus14 This [test](http://18.191.94.64/job/pygdf-prb/1/console) for #154 fails a few categorical tests and is using the Pandas 0.23.3. Is the newer version the cause? Should I downgrade Pandas for the tests?

Reply:
@mike-wendt Can you try using Pandas 0.20.3 and see if it succeeds with that? That's the version we locked to in our conda environment intentionally.

Reply:
@kkraus14 looks like it is working off of the `conda_environments/gdf_build.yml` there are no fixed versions but I have a `sed` command to update them for now as a temp workaround. Can see the output for the passing PR #154 test [here](http://18.191.94.64/job/pygdf-prb/2/console) and the now passing master test [here](18.191.94.64/job/pygdf-master/5/console)

Reply:
@yashv28 "make -j" with no core count arg will use all available cores.

@kkraus14 specified the right Pandas version and added link to the issue blocking newer versions